core/src/display_object/graphic.rs

```diff
@@ -65,13 +65,8 @@ impl<'gc> TDisplayObject<'gc> for Graphic<'gc> {
 
     fn world_bounds(&self) -> BoundingBox {
         // TODO: Use dirty flags and cache this.
-        let mut bounds = self.local_bounds();
-        let mut node = self.parent();
-        while let Some(display_object) = node {
-            bounds = bounds.transform(&*display_object.matrix());
-            node = display_object.parent();
-        }
-        bounds
+        self.local_bounds()
+            .transform(&self.local_to_global_matrix())
```

Comment:
I believe this entire method could be removed, instead using the base `DisplayObject::world_bounds`.